vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.cc

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+#include "bat/ads/internal/ml/ml_transformation_util.h"
+#include "bat/ads/internal/ml/transformation/transformation.h"
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(const PipelineEmbeddingInfo& info) {
```

Comment:
could you use `default` here?

Reply:
Resolved and aligned to better match the updates made to pipeline_info.cc